### DISCUSSION TITLE
Font size change

### DISCUSSION THREAD

**Discussion by macgarnigle** [nroduit/Weasis#561] [General]
I'd like to make the image overlay and DICOM information font bigger. I've adjusted the scaling, but it makes all the UI elements bigger as well. I was happy with the size of everything else at 100% just not the fonts.

**Comment by nroduit** [comment]
The views display a font size according to the [scaling](https://weasis.org/en/tutorials/theme/index.html#how-to-scale-the-user-interface) of the application. Overlay text is the same size as component text. Afterward, if the view is downsized or if there are several views, the text is smaller.

There is currently no way to change font size independently for image views.
